etc/Dockerfile.mod.pi

```diff
@@ -0,0 +1,50 @@
+ARG BASE_TAG
+ARG BASE_NAME
+ARG HUB_USER
+
+FROM ghcr.io/${HUB_USER}/${BASE_NAME}:${BASE_TAG}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install software-properties-common to get apt-add-repository
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    software-properties-common
+
+# Add the Raspberry Pi repository
+RUN apt-add-repository 'deb http://archive.raspberrypi.org/debian/ bullseye main'
+
+# Import the repository's signing key
+RUN wget -qO - https://archive.raspberrypi.org/debian/raspberrypi.gpg.key | apt-key add -
+
+# Update the package list
+RUN apt-get -y update
```

Comment:
Aren't we doing this up top?

Reply:
Again, I need to update here after adding to source list.